src/scripts/core/visualization-container.js

```diff
@@ -74,6 +74,12 @@
             return this.ctx;
         },
 
+        setVisibility: function (visible) {
+            var node = this.layer.node();
+
+            visible ? $(node).show() : $(node).hide();
```

Comment:
This (finally) makes sense. I've replaced the jquery code with javascript in my latest commit - you can see this in action at http://jsfiddle.net/q1psy5t3/3/.